src/us/kbase/workspace/database/mongo/MongoWorkspaceDB.java

```diff
@@ -1014,6 +1014,7 @@ public ResolvedWorkspaceID resolveWorkspace(
 			final Set<WorkspaceIdentifier> wsis,
 			final boolean allowDeleted,
 			final boolean allowMissing)
+			// TODO CODE don't think comm exception needs to be checked. Usually will just fail.
```

Comment:
Can you add a test for this?

Reply:
Not sure what you mean, this is just a comment. There's nothing to test.

Maybe you mean changing the communication exception to an unchecked exception when and if that happens? There's no way to test for that either, though...

Reply:
> TODO CODE don't think comm exception needs to be checked. Usually will just fail.

When I see a TODO comment, I try to ask "What is actionable here?" with the hopes of moving it to a ticketing system. For this one, I'd also ask how we can more certainty around the "I don't think" part.

Reply:
See above re TODOs. Re certainty, I need to look at how the exception is used in the code base, which is not something I'm going to get to immediately and is not important enough for a ticket.